src/corehost/common/pal.unix.cpp

```diff
@@ -170,6 +170,85 @@ bool pal::get_default_servicing_directory(string_t* recv)
     return true;
 }
 
+bool pal::get_local_shared_package_dir(pal::string_t* recv)
+{
+    recv->clear();
+    pal::string_t dir;
+    if (!pal::getenv("HOME", &dir))
+    {
+        struct passwd* pw = getpwuid(getuid());
+        if (pw && pw->pw_dir)
+        {
+            dir.assign(pw->pw_dir);
+        }
+    }
+    if (dir.empty())
+    {
+        return false;
+    }
+    append_path(&dir, _X(".dotnet"));
```

Comment:
I just moved it to [utils.cpp](https://github.com/dotnet/core-setup/pull/1330/files#diff-fe7492df9cc9bec21e24dcb50e04b9c0R256), as this is now platform independent